NOTICES.txt

```diff
@@ -173,7 +173,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 
 
->>> k8s.io/api-0.0.0-20180712090710-2d6f90ab1293
+>>> k8s.io/api-0.23.1
 
 Copyright [yyyy] [name of copyright owner]
```

Comment:
I see a lot of lines like this in `NOTICES.txt`, with placeholders for year and copyright owner. Should these be filled out?

I'll still approve, as the file has had these placeholders for at least 2 years.